samples/matter/template/boards/nrf54h20_suit_overlay.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable SUIT services
+CONFIG_SUIT=y
```

Comment:
I have few comments on that:
1. Why do we create nrf54h20_suit overlay that seems to enable only BT SMP? Can't SUIT be used with Matter OTA?
2. Why do we create such overlay at all, since we have `CONFIG_CHIP_DFU_OVER_BT_SMP` that is used by the clients to enable BT SMP? Wouldn't be better to modify the existing Kconfig to e.g. enable MCUBOOT or SUIT conditionally, depending on the platform used?

Reply:
1. Because it is only a base for SUIT. We need to check whether SUIT works with dfu_target to run OTA.
2. OK, that's a good point to edit the existing kconfig. I forgot that it belongs to us, not to Zephyr. I will move the configs there.